src/Orc.SupportPackage.Xaml/Orc.SupportPackage.Xaml.Shared/ViewModels/SupportPackageViewModel.cs

```diff
@@ -55,36 +59,19 @@ public class SupportPackageViewModel : ViewModelBase
             CreateSupportPackage = new TaskCommand(OnCreateSupportPackageExecuteAsync, OnCreateSupportPackageCanExecute);
             OpenDirectory = new Command(OnOpenDirectoryExecute, OnOpenDirectoryCanExecute);
 
-            SupportPackageFileTypes = new List<SupportPackageFileType>
+            SupportPackageFileSystemArtifacts = new List<SupportPackageFileSystemArtifact>();
```

Comment:
A bit more logging would be nice here. E.g. for each package content provider, log the type + tell that we are about to retrieve the artifacts